modules/node_modules/@ncigdc/containers/SmartSearchPage.js

```diff
@@ -94,6 +94,20 @@ class SmartSearchComponent extends React.Component {
           this.Error = null;
 
           this.setQuery();
+
+          document.querySelector('.btn-search-query').onclick = () => {
```

Comment:
According to this SO http://stackoverflow.com/a/37096563/482053 memory leaks from unattended event listeners aren't a problem anymore ?

Not worth blocking merge of a bug fix in any case, lgtm